about.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About | CC Global Summit 2015</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato&family=Poppins:wght@400;500;600;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <header class="main-header-about">
+    <nav class="social-media-nav">
+      <ul class="social-media">
+        <li class="social-list">
+          <a href="#"><img src="images/facebook (1).png" alt="facebook logo"></a>
+        </li>
+        <li class="social-list"><a href="#"><img src="images/twitter.png" alt="twitter logo"></a>
+        </li>
+        <li class="social-list">Spanish</li>
+        <li class="social-list">My Page</li>
+        <li class="social-list">logout</li>
+      </ul>
+    </nav>
+    <nav class="desktop-menu">
+      <a href="index.html"><img src="images/logo-conference.png" alt="logo culture days" width="130" height="51"></a>
+      <ul class="main-menu">
+        <li class="menu-list"><a href="about.html">About</a></li>
```

Comment:
- I think it's better to create a link to the index page instead of link to the same (about) page. Your code will be more professional and create a better UX.